src/core/socket.c

```diff
@@ -793,6 +796,11 @@ static void socket_dump(Unit *u, FILE *f, const char *prefix) {
                 }
         }
 
+        if (s->clear_socket)
+                fprintf(f,
+                        "%sClearSocket: %s\n",
+                         prefix, yes_no(s->clear_socket));
```

Comment:
you only show the field when it is off, but then it will always show as yes. Drop the conditionalization with `if` here (and maybe merge this into some other suitable fprintf() around here